SimMuon/MCTruth/python/SimMuFiltDouble_cff.py

```diff
@@ -0,0 +1,7 @@
+import FWCore.ParameterSet.Config as cms
+
+from SimMuon.MCTruth.SimMuFilter_cfi import SimMuFilter
+
+SimMuFilter.nMuSel = 2
```

Comment:
hi @kskovpen - to change parameters of a module, you need to clone it ( to avoid conflicts). 
eg, 
SimMuFilter2Mu = SimMuFilter.clone()
SimMuFilter2Mu.nMuSel=2